### PR TITLE
Expose experimental flags for GPU fallbacks

### PR DESCRIPTION
**Pull request by irfansharif**
Just for prototype purposes, for now -- will definitely change. Behind the scenes we may not even want to represent these as a single function definition.

### CODE REVIEW

File: modal/app.py [hunk 1]
```diff
@@ -460,7 +460,7 @@ def function(
         image: Optional[_Image] = None,  # The image to run as the container for the function
         schedule: Optional[Schedule] = None,  # An optional Modal Schedule for the function
         secrets: Sequence[_Secret] = (),  # Optional Modal Secret objects with environment variables for the container
-        gpu: GPU_T = None,  # GPU specification as string ("any", "T4", "A10G", ...) or object (`modal.GPU.A100()`, ...)
+        gpu: GPU_T = None,  # GPU specification as string ("any", "T4", "A10G", ...) or object `modal.GPU.A100()`, ...)
```
Comment (mwaskom) [inline modal/app.py:463]: Missing open parens?

Comment (irfansharif) [reply modal/app.py:463]: Oops cursor got too aggressive

Comment (mwaskom) [reply modal/app.py]: Lmao sure throw the ai under the bus

**Comment by mwaskom** [comment]
I wonder if we should be annoying and fire a warning or something when these flags are used … I feel like in the past we’ve had users adopt them and then complain when they change…

**Comment by irfansharif** [comment]
agreed but plz lemme be lazy for a bit (trying to speed run this)

**Comment by mwaskom** [comment]
image

**Comment by irfansharif** [comment]
@prbot approve

**Pull request merged**